model/Export/Stylesheet/AssetStylesheetLoader.php

```diff
@@ -51,15 +51,15 @@ public function loadAssetsFromAssetResource(string $link): ?array
 
             $stylesheetPath = $this->buildAssetPathFromPropertyName($property);
             try {
-                $cssFiles = $this->getFileSystem()->listContents($stylesheetPath);
+                $cssFiles = $this->getFileSystem()->listContents($stylesheetPath)->toArray();
                 foreach ($cssFiles as $key => $file) {
                     $cssFiles[$key]['stream'] = $this->getFileSystem()->readStream(
                         $stylesheetPath . DIRECTORY_SEPARATOR . $file['basename']
                     );
                 }
 
                 return $cssFiles;
-            } catch (FileNotFoundException $exception) {
+            } catch (FilesystemException $exception) {
```

Comment:
I don't really like that we bound to flysystem exception `League\Flysystem\FilesystemException` inside tao here and in some other places. Maybe it makes sense to create an internal tao exception and catch all Filesystem exceptions here `FileSystemWrapperTrait` and throw the tao exception instead?

Reply:
Thanks, great suggestion, I've updated PRs to catch a filesystem exception defined in our application